package.json

```diff
@@ -26,27 +26,27 @@
     "npm": "^10"
   },
   "devDependencies": {
-    "@eslint/compat": "^1.1.1",
+    "@eslint/compat": "^1.4.1",
     "@eslint/eslintrc": "^3.1.0",
-    "@eslint/js": "^9.11.1",
-    "@playwright/test": "^1.51.0",
+    "@eslint/js": "^9.39.1",
+    "@playwright/test": "^1.56.1",
     "@types/eslint__js": "^8.42.3",
-    "@types/node": "^22.5.5",
+    "@types/node": "^24.10.0",
     "allure-commandline": "^2.29.0",
-    "allure-playwright": "^3.0.0-beta.3",
-    "cross-env": "^7.0.3",
-    "dotenv": "^16.4.5",
-    "eslint": "^9.11.1",
+    "allure-playwright": "^3.4.2",
+    "cross-env": "^10.1.0",
+    "dotenv": "^17.2.3",
+    "eslint": "^9.39.1",
     "eslint-plugin-no-only-tests": "^3.3.0",
-    "globals": "^15.9.0",
+    "globals": "^16.5.0",
     "mkdirp": "^3.0.1",
     "prettier": "^3.3.2",
-    "rimraf": "^5.0.7",
-    "typescript": "^5.6.2",
-    "typescript-eslint": "^8.7.0"
+    "rimraf": "^6.1.0",
+    "typescript": "^5.9.3",
+    "typescript-eslint": "^8.46.3"
   },
   "dependencies": {
     "date-fns": "^4.1.0",
-    "uuid": "^11.1.0"
+    "uuid": "^13.0.0"
```

Comment:
You can remove this and wherever you're generating a uuid you can use similar to:

`const { randomUUID } = require('crypto')`

then

`const random = randomUUID()`